packages/image_picker/lib/image_picker.dart

```diff
@@ -58,13 +58,15 @@ class ImagePicker {
 
   static Future<File> pickVideo({
     @required ImageSource source,
+    double maxDuration
```

Comment:
Also: would it be better if maxDuration is actually of type `Duration`? Currently it is unclear what the unit for durations is. Seconds? Minutes? Hours?

Reply:
I will add a note that the maxDuration is in Seconds.